CONTRIBUTING.md

```diff
@@ -99,27 +104,42 @@ Example of a module directory:
     └── util
         └── get_week.py
     ```
-  * The respective [Wiki](https://github.com/NREL/HPC/wiki) directory featuring verbose explanations:
-    ```bash
-    plexos-quickstart
-    ├── initial-session.md
-    ├── README.md  # The "home page" of the module which introduces, links to, and structures neighboring pages.
-    ├── run-magma.md
-    ├── run-plexos.md
-    └── setup-plexos.md
-    ```
-
-### Markdown content and structure
-* The intent of training or tutorial materials is to show someone who does not know how to do what you do how to do it. Err on the side of verbosity.
-
-* Where you can assume that a user should have worked through training materials conceptually prior to the current ones, cross-reference them (mention and hyperlink). This also applies to information on the HPC website. In general, where you can refer someone to well crafted information online that's more extensive or well constructed than you have time or space to write, do so through a hyperlink.
-
-* **Images and other reusable non-textual files go in `assets/`** in the root of both the repository and the [Wiki](https://github.com/NREL/HPC/wiki) of this repository, so as to be easily and mutually referred to throughout various pages. The image can then be linked to anywhere with markdown like `![alt text](/assets/name_of_image.png)`, If an image can replace lots of text, by all means include it. However, don't take terminal screenshots when `inline code` or a codeblock will suffice.
+* **Images and other reusable non-textual files go in `assets/`** in the root of the repository, so as to be easily and mutually referred to throughout various pages. The image can then be linked to anywhere with markdown like `![alt text](/assets/name_of_image.png)`, If an image can replace lots of text, by all means include it. However, don't take terminal screenshots when `inline code` or a codeblock will suffice.
```

Comment:
**Images and other reusable non-textual files go in `assets/`** in the root of the repository, to be easily and mutually referred to throughout various pages. The image can then be linked to anywhere with markdown as `![alt text](/assets/name_of_image.png)`, If an image can replace lots of text, by all means, please include it. However, don't take terminal screenshots when `inline code` or a code block will suffice.